### DISCUSSION TITLE
site

### DISCUSSION THREAD

**Discussion by 1sosed9** [community/community#32952] [Other Feature Feedback, Questions, & Ideas]
Как мне получить ссылку, которая выводит, как будет выглядеть сайт?

**Comment by queenofcorgis** [comment]
_Translation: How do I get a link that displays what the site will look like?_

**Comment by jfullstackdev** [comment]
it's at the Settings of the repo once you enabled it there, like this

![image](https://user-images.githubusercontent.com/47092464/199152877-be227f1e-870d-4f80-b8dc-9ebf51f4637a.png)
